src/xcode/ENA/ENA/Source/Services/__tests__/Mocks/MockTestStore.swift

```diff
@@ -11,6 +11,12 @@ final class MockTestStore: Store, PPAnalyticsData {
 	var firstPlaybookExecution: Date?
 	var lastBackgroundFakeRequest: Date = .init()
 	var hasSeenBackgroundFetchAlert: Bool = false
+	private var _exposureDetectionDate: Date?
+	var exposureDetectionDate: Date? {
+		get { _exposureDetectionDate as Date? ??
+			enfRiskCalculationResult?.calculationDate }
+		set { _exposureDetectionDate = newValue }
+	}
```

Comment:
Following your argumentation to have an independent softRateLimit date to the `calculationDate`, i am not sure if we should take it as the default value.

Reply:
Yes, you are right. I was preoccupied because the original idea was a true rate limit, and I wanted a smooth transition. But for a logger this is not necessary.